django_oikotie/xml_models/apartment.py

```diff
@@ -214,7 +214,8 @@ def to_etree(self) -> etree._Element:
         return element
 
     def format_description(self) -> str:
```

Comment:
Since this function can now return `None`, I think the return type should be `Optional[str]`.